packages/office-addin-unit-test/src/officeJSMock.ts

```diff
@@ -0,0 +1,150 @@
+import { usageDataObject } from "./defaults";
+
+export class OfficeJSMock {
+  constructor(object?: OfficeObject) {
+    this.properties = new Map<string, OfficeJSMock>();
+    this.loaded = false;
+    this.resetValue(undefined);
+    if (object) {
+      this.populate(object);
+    }
+  }
+
+  // Adds a function to OfficeJSMock
+  addMockFunction(methodName: string, functionality?: Function) {
+    this[methodName] = functionality ? functionality : function () {};
+  }
+
+  // Adds a OfficeJSMock to OfficeJSMock
+  addMockObject(objectName: string) {
+    const officeJSMock = new OfficeJSMock();
+    officeJSMock.isMockObject = true;
+    this.properties.set(objectName, officeJSMock);
+    this[objectName] = this.properties.get(objectName);
+  }
+
+  // load of Office.js API
+  load(property: string) {
```

Comment:
What does "load of Office.js API" mean?  Took me a while to figure out this was the replacement for the load call from the original office.js API.

Also don't see a need to have a nested private function instead of having it all directly here.

Reply:
Yeah, this is the replacement of the original load from office-js API. I will work on making it more clear in the comments.

About the nested private function, I think I still like having it explicitly there to make it easier to understand what the piece of code is doing (handling multiple properties). Given that `load` has to be the name of this function, having the private one with a more descriptive name seems more clear to me.

Reply:
The load from Office.js does the same handling of multiple properties so it matches that.

I just dislike having to dig through many layers of code to find the meat of an operation.